### ISSUE TITLE
Test data seen during training - correct results?

### ISSUE THREAD

**Issue by adrianastan** [AryaAftab/LIGHT-SERNET#7]
Hi,

I just noticed in your code that you are using the [test data from the CV fold as validation data](https://github.com/AryaAftab/LIGHT-SERNET/issues/new) and save the best model based on the validation accuracy. This is sort of cherry picking the model. Do you by any chance have updated results where you do not use the test set during training?

Thanks,
Adriana
